saleor/graphql/order/types.py

```diff
@@ -654,6 +654,11 @@ class OrderLine(ModelObjectType[models.OrderLine]):
     total_price = graphene.Field(
         TaxedMoney, description="Price of the order line.", required=True
     )
+    undiscounted_total_price = graphene.Field(
+        TaxedMoney,
+        description=("Price of the order line without applied an order line discount."),
```

Comment:
👮‍♂️ 🚓 
```suggestion
        description="Price of the order line without applied an order line discount.",
```

Reply:
Also maybe 
`Price of the order line without discounts.`

Reply:
I was trying to stay "close" to the other descriptions, but I'm fine with changing it.